shared/src/main/scala/pl/metastack/metarx.scala

```diff
@@ -129,4 +129,11 @@ package object metarx extends OptImplicits {
     def <(other: ReadChannel[T]): ReadChannel[Boolean] = other.map(value < _)
     def <=(other: ReadChannel[T]): ReadChannel[Boolean] = other.map(value <= _)
   }
+
+  implicit class ReadChannelStringExtensions(rch: ReadChannel[String]) {
+    def +(other: ReadChannel[String]): ReadChannel[String] = rch.zipWith(other)(_ + _)
+    def +(value: String): ReadChannel[String] = rch.map(_ + value)
+
+    def length() = rch.map(_.length())
```

Comment:
I think we should limit ourselves to operators. Otherwise we'd have to include many other functions as well such as `substr()`.